x-pack/plugins/security_solution/public/common/containers/alerts/use_alert_prevalence.ts

```diff
@@ -33,6 +33,7 @@ interface UserAlertPrevalenceResult {
   alertIds?: string[];
 }
 
+// TODO: MOVE TO FLYOUT FOLDER
```

Comment:
Consider adding the issue link for the remaining work in all these `// TODO` comments.

Reply:
I actually already have an entry in this overall [ticket](https://github.com/elastic/security-team/issues/7462) that takes care of that!